rascal-vscode-extension/.vscode/launch.json

```diff
@@ -49,6 +56,30 @@
 				"${workspaceFolder}/out/test/**/*.js"
 			],
 			"preLaunchTask": "npm: pretest"
-		}
+		},
+		{
```

Comment:
do we need to copy this over to this launch.json? it's already in rascal-lsp one? won't this cause duplicate entries in the list of launch configureations?

Reply:
I wanted to create a single button launch for the entire second level. That works sometimes, and sometimes it does not. I don't understand the difference.